kernelci/test.py

```diff
@@ -79,8 +79,11 @@ def get_params(bmeta, target, plan_config, storage):
     file_server_resource = bmeta['file_server_resource']
     job_px = file_server_resource.replace('/', '-')
     url_px = file_server_resource
-    job_name = '-'.join([job_px, dtb or 'no-dtb',
```

Comment:
Again, there's no need to change anything in `kernelci.test`.  The job name length is a LAVA limitation and it can just be truncated in `kernelci.lab.lava`.